packages/grape-web-client/src/actions/channelMembersInvite.js

```diff
@@ -29,14 +29,18 @@ export function removeFromChannelMembersInvite(user) {
   }
 }
 
-export const searchUsersToInvite = value => (dispatch, getState) => {
+export const searchUsersToInvite = searchText => (dispatch, getState) => {
   dispatch({
     type: types.FILTER_CHANNEL_MEMBERS_INVITE,
-    payload: value,
+    payload: searchText,
```

Comment:
maybe `search` instead of `searchText` because `searchOverview` also uses `search`